apps/web/src/components/notification-center/NotificationRenewal/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState, type ReactElement } from 'react'
+import { Alert, Box, Button, Typography } from '@mui/material'
+import useSafeInfo from '@/hooks/useSafeInfo'
+import CheckWallet from '@/components/common/CheckWallet'
+import { useNotificationsRenewal } from '@/components/settings/PushNotifications/hooks/useNotificationsRenewal'
+import { useIsNotificationsRenewalEnabled } from '@/components/settings/PushNotifications/hooks/useNotificationsTokenVersion'
+
+const NotificationRenewal = (): ReactElement => {
+  const { safe, safeLoaded } = useSafeInfo()
+  const [isRegistering, setIsRegistering] = useState(false)
+  const { renewNotifications, needsRenewal, numberChainsForRenewal } = useNotificationsRenewal()
+  const isNotificationsRenewalEnabled = useIsNotificationsRenewalEnabled()
+
+  if (!needsRenewal || !isNotificationsRenewalEnabled) {
+    // No need to renew any Safe's notifications
+    return <></>
+  }
+
+  const handeSignClick = async () => {
+    setIsRegistering(true)
+    await renewNotifications()
+    setIsRegistering(false)
+  }
+
+  const message = `We’ve upgraded your notification experience. Sign ${safeLoaded || numberChainsForRenewal < 2 ? 'the message' : `${numberChainsForRenewal} messages`} now to keep receiving important updates seamlessly.`
+
+  return (
+    <>
+      <Alert severity="warning">
+        <Typography
+          variant="body2"
+          sx={{
+            fontWeight: 700,
+            mb: 1,
+          }}
```

Comment:
```suggestion
          fontWeight={700}
          mb={1}
```

Typography and Box allow this neater syntax. Otherwise, larger CSS must be in a CSS module.

Reply:
Done